extensions/spark/kyuubi-spark-connector-tpcds/pom.xml

```diff
@@ -213,4 +213,29 @@
         <outputDirectory>target/scala-${scala.binary.version}/classes</outputDirectory>
         <testOutputDirectory>target/scala-${scala.binary.version}/test-classes</testOutputDirectory>
     </build>
+
+    <profiles>
+        <profile>
+            <id>gluten</id>
```

Comment:
Can we make a new module for Gluten IT ? Includes tpcds and tpch related change.

Reply:
good idea, i will implement this.